src/state/apis/nft/types.ts

```diff
@@ -0,0 +1,36 @@
+import type { AssetId, ChainId } from '@shapeshiftoss/caip'
+
+import type { MediaUrl } from '../zapper/validators'
+
+export type NftCollectionItem = {
+  // The AssetId of the collection, which doesn't exist as an assetSlice asset yet (https://github.com/shapeshift/web/pull/4555)
+  // Note, depending on the endpoint being hit, we may or may not be able to get an actual collection AssetId
+  id: AssetId | null
```

Comment:
benefit to using assetId format over just the address or empty string?

Reply:
then we don't need to worry about the `fromAssetId` or knowing if we have data or not, it's either the id, or empty string

Reply:
I guess `CollectionItem` may be a confusing terminology here, this refers to a collection contract address, which is why we make it an actual `AssetId` - for a single NFT we only need an `id`

Reply:
e1d706332d5e50f9e888f00a37cfa0ef1fa5dd34